src/routes.rs

```diff
@@ -227,8 +251,72 @@ pub async fn index(request: HttpRequest, identity: Identity) -> impl Responder {
             INDEX_CONTENTS.to_owned()
         }
     };
+    let final_contents = contents.replace("{{ themepath }}", &CONFIG.highlight.themepath);
```

Comment:
That new variable isn't really necessary, you can chain `replace` after the block (`{ ... }.replace(...);`).